english/prostate/prostate.md

```diff
@@ -0,0 +1,5 @@
+“It’s a damn shame when a man can’t get a few hours of sleep without having to get up to pee,” you think as you put on your slippers and shuffle to the bathroom.  Ugh, that padded vinyl toilet seat is cold.  You realize it’s been a while since you had your prostate checked.  Do you:
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.